pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -15,10 +15,10 @@ stdenv.mkDerivation rec {
     owner = "openshift";
     repo = "origin";
     rev = "v${version}";
-    sha256 = "0zw8zb9c6icigcq6y47ppnjnqyghk2kril07bapbddvgnvbbfp6m";
+    sha256 = "0qvyxcyca3888nkgvyvqcmybm95ncwxb3zvrzbg2gz8kx6g6350v";
   };
 
-  buildInputs = [ go which ];
+  buildInputs = [ go which removeReferencesTo makeWrapper ];
```

Comment:
Was buildGoPackage an option here? This will remove the go dependency automatically.